<launch>

    <!-- 启动手柄原始节点 -->
    <node name="joy_node" pkg="joy" type="joy_node" output="screen" respawn="true"/>

    <!-- 启动塔克手柄遥控机器人节点，适合所有车型 -->
    <node name="teleop_joy" pkg="joy" type="xtark_teleop_joy.py" output="screen" respawn="true">
        <param name="x_speed_scale"  value="0.8" />
        <param name="y_speed_scale"  value="0.8" />
        <param name="w_speed_scale"  value="1.0" />
    </node>
    
</launch>
